components/collection/unlockable/UnlockableContainer.vue

```diff
@@ -1,6 +1,6 @@
 <template>
   <div class="unlockable-container">
-    <Loader v-model="isLoading" :minted="justMinted" />
+    <Loader v-model:value="isLoading" :minted="justMinted" />
```

Comment:
or we can change in `Loader` prop `value` to `modelValue` so we won't need to update each `<Loader />` component call